scripts/generate-app-config/generateAppConfigCli.js

```diff
@@ -0,0 +1,15 @@
+const commander = require('commander');
+const generate = require('./generateAppConfig');
+
+const packageJson = require('../../package.json');
+const loadDefaultSiteConfig = require('./loadDefaultSiteConfig');
+
+
+// Parse process arguments
+commander
+  .version(packageJson.version)
+  .option('--config <path>', 'The site config.', undefined)
```

Comment:
Do you think it would be valuable to elaborate the site-config loading pattern here since we use `loadDefaultConf` and